tools/csv-merger/csv-merger.go

```diff
@@ -80,10 +79,8 @@ var (
 )
 
 type templateData struct {
-	RookOperatorImage      string
-	RookOperatorCsvVersion string
-	OcsOperatorCsvVersion  string
-	OcsOperatorImage       string
+	OcsOperatorCsvVersion string
```

Comment:
I am wondering if ocs csv will be the sole csv we will be generating do we really need this csv-merger tool anymore? as there are no more csv's to merge, it's just the ocs csv now.

Reply:
Agreed, though I feel the tool is still needed to make some changes to the csv at different places. Maybe we can rename it to something else if needed, but removing it won't work I think.